std_msgs/Header header
sensor_msgs/PointCloud2 cloud_source
sensor_msgs/PointCloud2 cloud_target
int32[] index_source
int32[] index_target